core/layers/svc-core-proxy/extension/server/serve.go

```
@@ -0,0 +1,110 @@
+}
+
+type Config struct {
+	Port            string
+	EnableExtension bool
+}
+
+// Start begins running the sidecar
+func Start(port string, server *HttpServer, enableExtension bool) {
+	if enableExtension {
+		log.Println("Starting the server in background")
+		go startHTTPServer(port, server)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)